WordPress/WordPressTest/SiteCreation/SiteCreationRotatingMessageViewTests.swift

```diff
@@ -0,0 +1,73 @@
+import XCTest
+@testable import WordPress
+
+class SiteCreationRotatingMessageViewTests: XCTestCase {
+    private let testMessages = [
+        "Test Message 1",
+        "Test Message 2",
+        "Test Message 3"
+    ]
+
+    private var rotatingMessageView: SiteCreationRotatingMessageView?
+
+    override func setUp() {
+        super.setUp()
+
+        self.rotatingMessageView = SiteCreationRotatingMessageView(messages: testMessages,
+                                                                   iconImage: UIImage())
+    }
+
+    override func tearDown() {
+        self.rotatingMessageView = nil
+
+        super.tearDown()
+    }
+
+    /// Test to make sure the statusLabel text and accesibility labels are being set correctly
+    func testSiteCreationRotatingMessageView_StatusUpdate() {
+        let message = "This is a test!"
+        rotatingMessageView?.updateStatus(message: message)
+
+        XCTAssertEqual(rotatingMessageView?.statusLabel.text, message)
+        XCTAssertEqual(rotatingMessageView?.statusLabel.accessibilityLabel, message)
```

Comment:
This line is failing — since we removed manually setting the `accessibilityLabel`. Let's remove it!